src/Actions/Contexts/ElementContext.cs

```diff
@@ -39,9 +39,8 @@ public class ElementContext : IDisposable
         /// <param name="element"></param>
         public ElementContext(A11yElement element)
         {
-            if (element == null) throw new ArgumentNullException(nameof(element));
+            this.Element = element ?? throw new ArgumentNullException(nameof(element));
```

Comment:
There are lots other places (mainly constructors) where this pattern occurs, but since this is the only one that VS flagged, it's the only one fixed in this PR.